packages/@react-stately/table/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.6.2",
+    "@react-aria/utils": "^3.11.2",
```

Comment:
I forgot about this one. We should talk on monday about what we want to do here. I only used the one from `@react-aria/utils` because I got a warning to use it.

Reply:
Info on this one, we're leaving the circular in for the moment so we don't add an export to stately/utils that we don't really want in stately/utils. We'll be addressing this in followup. We think this particular case won't cause an issue or bugs because we've apparently accidentally done this before. We do want to remove it soon, but again, not necessary for getting this work into the hands of teams that need to try it out.